docs/headless-chrome.md

```diff
@@ -1,10 +1,33 @@
 # Running Lighthouse using headless Chrome
 
-For now, we recommend running Chrome with xvfb. See below.
+## CLI (headless)
+
+> **Note**: Headless Chrome still has a few bugs to work out. For example, [network emulation](https://bugs.chromium.org/p/chromium/issues/detail?id=728451) is not supported yet.
```

Comment:
can we remove this caveat now? :D

might be worth still noting there may be some graphics-related perf differences

Reply:
👍 . offline and throttling are implemented.  we can nuke the note.

there should probably be a list somewhere which documents known differences between headless and headful. but there isn't one yet.

Reply:
ping @ebidel